File: Tugas2/python/disk.py
class disk():
    __type = ""
    __capacity = ""
    __pricep = 0

    # construct
    def __init__(self, type = "", capacity = "", pricep = 0):
        self.__type = type
        self.__capacity = capacity
        self.__pricep = pricep

    # set dan get
    def settype(self, type):
        self.__type = type

    def gettype(self):
        return self.__type
    
    def setcapacity(self, capacity):
        self.__capacity = capacity
    
    def getcapacity(self):
        return self.__capacity

    def setpricep(self, pricep):
        self.__pricep = pricep
    
    def getpricep(self):
        return self.__pricep

    # output
    def printdisk(self):
        print("type : " + str(self.__type))
        print("capacity : " + str(self.__capacity))
        print("price : " + str(self.__pricep))

File: Tugas1/disk.cpp
#include <iostream>
#include <string>

using namespace std;

class disk{
	private:
		string type;
        string capacity;
        int priced;
	
	public:
		//Construct
		disk(){}
		
		disk(string type, string capacity, int priced){
			this->type = type;
			this->capacity = capacity;
			this->priced = priced;
		}
		
		//Set dan Get
		void settype(string type)
        {
            this->type = type;
        }
		string gettype(){
			return this->type;
		}
		
		void setcapacity(string capacity){
			this->capacity = capacity;
		}
		string getcapacity(){
			return this->capacity;
		}
		
		void setpriced(int priced){
			this->priced = priced;
		}
		int getpriced(){
			return this->priced;
		}
        
        void printdisk(){
            cout << "Tipe Disk      : " << gettype() << endl;
            cout << "Kapasitas Disk : " << getcapacity() << endl;
            cout << "Harga Disk     : " << getpriced() << endl;
        }		
			
		
		~disk(){
		}
};